test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchGlooCpu

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Please modify the pre-existing `testPyTorchGloo` script by adding conditions to determine whether the test should be run with CPU args or with GPU args, rather than adding a whole new wrapper script around `dlbs/python/pytorch_benchmarks/benchmarks.py` just to test with CPU.